scripts/output/testing/main_data/Peru.csv

```diff
@@ -1,744 +1,751 @@
+Peru,2021-05-25,,14162,0.095,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-26,,15865,0.093,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-27,,15166,0.092,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-28,,12715,0.091,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-29,,9938,0.09,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-30,,2958,0.089,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-05-31,,15395,0.086,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,
+Peru,2021-06-01,12911851,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-02,12969305,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-03,13025140,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-04,13076134,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-05,13132238,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-06,13173460,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
+Peru,2021-06-07,13188675,,0.0,tests performed,https://datos.ins.gob.pe/dataset/dataset-de-pruebas-moleculares-del-instituto-nacional-de-salud-ins,National Institute of Health,Ministerio de Salud via https://github.com/jmcastagnetto/covid-19-peru-data
```

Comment:
Not sure how to fill the positive rates!
Maybe opening an issue in the source, so that all the historical data is included and we wouldn't need to attach.